### ISSUE TITLE
Tried to run "force whoami" and got an error

### ISSUE THREAD

**Issue by arufian** [ForceCLI/force#143]
I have tried both with compiled source and the latest precompiled binary in Mac OSX 10.9.5, and got this error below:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x24313]

goroutine 16 [running]:
runtime.panic(0x3def80, 0x67f804)
    /tmp/build_f8ba0f4168b936fea1eb59aef85c032b/go/src/pkg/runtime/panic.c:279 +0xf5
main.(*Force).Whoami(0x0, 0x220871a700, 0x0, 0x0)
    /app/vendor/lib/go/src/src/github.com/heroku/force/force.go:783 +0xf3
main.runWhoami(0x67ca80, 0x208716020, 0x0, 0x0)
    /app/vendor/lib/go/src/src/github.com/heroku/force/whoami.go:20 +0x2b
main.main()
    /app/vendor/lib/go/src/src/github.com/heroku/force/main.go:49 +0x2aa

goroutine 19 [finalizer wait]:
runtime.park(0x6d180, 0x683158, 0x681ba9)
    /tmp/build_f8ba0f4168b936fea1eb59aef85c032b/go/src/pkg/runtime/proc.c:1369 +0x89
runtime.parkunlock(0x683158, 0x681ba9)
    /tmp/build_f8ba0f4168b936fea1eb59aef85c032b/go/src/pkg/runtime/proc.c:1385 +0x3b
runfinq()
    /tmp/build_f8ba0f4168b936fea1eb59aef85c032b/go/src/pkg/runtime/mgc0.c:2644 +0xcf
runtime.goexit()
    /tmp/build_f8ba0f4168b936fea1eb59aef85c032b/go/src/pkg/runtime/proc.c:1445
```

anyone in here got the same issue like mine ?

**Comment by cloudconversion** [comment]
Yes, I get the same error on most CLI functions

**Comment by dcarroll** [comment]
Jon, did you resolve this issue?

**Comment by cloudconversion** [comment]
No, it's still an issue

On Thu, Jun 25, 2015 at 10:58 AM, Dave Carroll <notifications@github.com>
wrote:

> Jon, did you resolve this issue?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/heroku/force/issues/143#issuecomment-115323359>.
>



-- 
Jon Jessup
CEO | Cloud Conversion, Inc.
jon@cloudconversion.com  |  435-901-2130
http://www.CloudConversion.com

**Comment by donmac23** [comment]
I get the same kind of error, but on Windows 7. Have tried it with both 32 bit and 64 bit binaries. Only command I can run successfully is login.

**Comment by cwarden** [comment]
It looks like this was fixed in v0.22.65.
